app/main/views/services.py

```diff
@@ -246,6 +245,77 @@ def update_section_submission(service_id, section):
         )
 
 
+@main.route('/submission/g-cloud-7/create', methods=['GET'])
+@login_required
+@flask_featureflags.is_active_feature('CREATE_SERVICE_PAGE')
+def start_new_draft_service():
+    """
+    Page to kick off creation of a new (G7) service.
+    """
+    template_data = main.config['BASE_TEMPLATE_DATA']
+
+    lots = new_service_content.get_question('lot')
+    lots['type'] = 'radio'
+    lots['name'] = 'lot'
+    lots.pop('error', None)
+
+    # errors if they exist
+    error, errors = request.args.get('error', None), []
+    if error:
+        lots['error'] = error
+        errors.append({
+            "input_name": lots['name'],
+            "question": lots['question']
+        })
+
+    return render_template(
+        "services/create_new_draft_service.html",
+        errors=errors,
+        **dict(template_data, **lots)
+    ), 200 if not errors else 400
+
+
+@main.route('/submission/g-cloud-7/create', methods=['POST'])
+@login_required
+@flask_featureflags.is_active_feature('CREATE_SERVICE_PAGE')
+def create_new_draft_service():
+    """
+    Hits up the data API to create a new draft (G7) service.
+    """
+    lot = request.form.get('lot', None)
+
+    if not lot:
+        return redirect(
+            url_for(".start_new_draft_service", error="Answer is required")
+        )
+
+    # hard-coding in a framework slug :/
```

Comment:
@robyoung says it's not worth doing this, and I think he's right.

If we allow passed-in parameters then we have to validate them, and currently there's only one possibility that we would allow anyway and we already know what that possibility looks like.